proxy/core/src/main/scala/io/cloudstate/proxy/eventing/GooglePubsubEventing.scala

```diff
@@ -179,9 +173,8 @@ class GCPubsubEventingSupport(config: Config, materializer: ActorMaterializer) e
 
   // Create the gRPC clients used to communicate with Google Pubsub
   final val (subscriberClient, publisherClient) = {
-    implicit val m = materializer
-    implicit val s = materializer.system
-    implicit val d = s.dispatcher
+    implicit val m = Materializer.matFromSystem
```

Comment:
Why isn't this `Materialize(system)`?

Reply:
After taking another look at Materializer, good point.